jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/SizeLimitHandler.java

```diff
@@ -119,7 +120,7 @@ public SizeLimitResponseWrapper(Request request, Response wrapped)
                 @Override
                 public HttpField onAddField(HttpField field)
                 {
-                    if (field.getHeader().is(HttpHeader.CONTENT_LENGTH.asString()))
+                    if (HttpHeader.CONTENT_LENGTH.is(field.getName()))
```

Comment:
It is more efficient to do an enum comparison rather than a string comparison:
```suggestion
                    if (field.getHeader() == HttpHeader.CONTENT_LENGTH)
```

Reply:
done